server.test.js

```diff
@@ -30,17 +35,99 @@ describe('Games', () => {
     // write a beforeEach hook that will populate your test DB with data
     // each time this hook runs, you should save a document to your db
     // by saving the document you'll be able to use it in each of your `it` blocks
+    const newGame = new Game({
+      title: 'fifa 2018',
+      genre: 'soccer',
+      releaseDate: 'JAN 2018'
+
+    });
+    newGame.save((err, savedGame) =>{
+      if(err){
+        console.log(err);
+        done();
+      }
+      GameId = savedGame._id.toString();
+      done();
+    })
+
   });
   afterEach(done => {
     // simply remove the collections from your DB.
+    Game.remove({}, err => {
+      if (err) console.log(err);
+      done();
+    });
   });
 
   // test the POST here
+  describe(` [POST]  /api/game/create`, () => {
+    it('should save the game to the db', done =>{
+      chai.request(server)
+      .post('/api/game')
+      .send({ name: 'fifa2019', genre: 'soccer', releaseDate: 'Jan 2019'})
+      .then(response => {
+
+done();
+      })
+      .catch(err =>{
+        throw err;
+      });
+      it('should fail if no name or genre is provided', (done) =>{
+        chai
+        .request(server)
+        .post('/api/game/create')
+        .send({ name: 'fifa2019',  releaseDate: 'Jan 2019'})
+        .then(response => {
+          expect(reponse.status).to.equal(200);
+          expect(response.body).to.include(name,genre,releaseDate);
+          done();
+        })
+        .catch(err =>{
+          throw err;
+          done();
+        });
+      
+    });
+  });
 
   // test the GET here
+  describe(`[GET] /api/game/get`, () =>{
+    it('should display all lists of games in db', done => {
+      chai
+      .request(server)
+      .get('/api/game/get')
+      .then(response =>{
+expect(response.status).to.equal(200);
+expect(_id).to.equal(bandId);
+        done();
+      })
+      .catch(err =>{
+        throw err;
+      })
+    })
+it('should fail if a bad request is made', () =>{
+
+})
+  })
 
   // Test the DELETE here
+  describe(`[DELETE] /api/game/destroy/:id`, ()  => {
+    var url = 'http://localhost:5050';
+  
+      it('should return status 200 after DELETING a game', done => {
+       chai.request(url)
+          .delete(`/api/game/destroy/ ${gameId}`)
+          .end((err, response) => {
+              if (err) {
+                throw err;
+              }
```

Comment:
One convention is to write your if statement on one line to keep things short and concise.  You don't have to follow the convention, but I do think it cleans up your code and makes it concise and clear.  For example, you could write lines 121 through 123 like this in one line:
```if (err) throw eww;```